src/main/res/values/strings.xml

```diff
@@ -513,18 +511,18 @@ limitations under the License.
     <string name="voice_pace_per_kilometer">%1$s per kilometer</string>
     <string name="voice_pace_per_mile">%1$s per mile</string>
     <string name="voice_template">%1$s in %2$s at %3$s</string>
-    <string name="voice_speed_lap">Lap speed of %1$s</string>
-    <string name="voice_pace_lap">Lap time of %1$s</string>
+    <string name="voice_speed_lap">%1$s avg. lap speed</string>
+    <string name="voice_pace_lap">%1$s lap time</string>
     <plurals name="voiceTotalDistanceKilometers">
         <item quantity="one">total distance 1 kilometer</item>
         <item quantity="other">total distance %1$.2f kilometers</item>
     </plurals>
     <plurals name="voiceTotalDistanceMiles">
-        <item quantity="one">total distance 1 mile</item>
-        <item quantity="other">total distance %1$.2f miles</item>
+        <item quantity="one">total distance, 1 mile</item>
+        <item quantity="other">total distance, %1$.2f miles</item>
     </plurals>
     <!-- Waypoint Type -->
-    <string name="waypoint_type_atm">ATM</string>
+    <string name="waypoint_type_atm">Cash machine</string>
```

Comment:
It's an ATM in most places, right?

Reply:
It is in the US, whereas more people understand "cash machine" is my thinking.
It is possible to have both en_US and en_UK translations, but alas.